packages/client/src/sql/index.ts

```diff
@@ -31,6 +31,30 @@ export type SQLQueryParams<T = any[]> = {
   responseType?: 'json' | 'array';
 };
 
+export type SQLBatchQuery = {
+  /**
+   * The SQL statements to execute.
+   */
+  statements: {
+    /**
+     * The SQL statement to execute.
+     */
+    statement: string;
+    /**
+     * The parameters to pass to the SQL statement.
+     */
+    params?: any[];
+  }[];
+  /**
+   * The consistency level to use when executing the queries.
+   */
+  consistency?: 'strong' | 'eventual';
```

Comment:
What's the default? Should that info be included in the comment here? Same question for `responseType`